### ISSUE TITLE
get without arguments stack trace

### ISSUE THREAD

**Issue by virtuald** [batfish/batfish#211]
```
batfish> get 
java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.LinkedList.checkElementIndex(LinkedList.java:555)
	at java.util.LinkedList.get(LinkedList.java:476)
	at org.batfish.client.Client.get(Client.java:1224)
	at org.batfish.client.Client.processCommand(Client.java:2040)
	at org.batfish.client.Client.processCommand(Client.java:1985)
	at org.batfish.client.Client.runInteractive(Client.java:2319)
	at org.batfish.client.Client.run(Client.java:2272)
	at org.batfish.allinone.AllInOne.run(AllInOne.java:117)
	at org.batfish.allinone.Main.main(Main.java:6)
```

Maybe it could enumerate the things that one can get?

**Comment by virtuald** [comment]
Speaking of which, `list-questions` doesn't work either:

```
batfish> list-questions
exception: org.codehaus.jettison.json.JSONException: JSONObject["questionlist"] is not a JSONArray.
	at org.codehaus.jettison.json.JSONObject.getJSONArray(JSONObject.java:493)
	at org.batfish.client.BfCoordWorkHelper.listQuestions(BfCoordWorkHelper.java:957)
	at org.batfish.client.Client.listQuestions(Client.java:1753)
	at org.batfish.client.Client.processCommand(Client.java:2079)
	at org.batfish.client.Client.processCommand(Client.java:1985)
	at org.batfish.client.Client.runInteractive(Client.java:2319)
	at org.batfish.client.Client.run(Client.java:2272)
	at org.batfish.allinone.AllInOne.run(AllInOne.java:117)
	at org.batfish.allinone.Main.main(Main.java:6)
```

**Comment by virtuald** [comment]
... and there are other commands that don't check the number of parameters.

* set-loglevel
* set-environment
* test

... etc.

**Comment by kidsbear** [comment]
Implemented by PR#217, close for now.

**Comment by dhalperi** [comment]
@kidsbear Normally, don't close bugs until after the PR is merged.
